Pipfile

```diff
@@ -14,7 +14,7 @@ SQLAlchemy = "*"
 Werkzeug = "*"
 Flask-Migrate = ">=2.3.1"
 Flask-Script = "*"
-marshmallow = "==3.0.0rc6"
+marshmallow = ">=3.0.0rc7"
```

Comment:
I have concerns about that version. Yesterday I updated my marshmallow to rc7 and experienced 500 error related to `partial` on load (I don't use `partial` at all, so that's probably a bug). rc6 seems more stable